### DISCUSSION TITLE
How to use !elements keyword in DSL

### DISCUSSION THREAD

**Discussion by vanyarock01** [structurizr/java#340] [Q&A]
### Question

I want to set custom style (color eg) for elements in the group. So, i found this page https://docs.structurizr.com/dsl/language#elements and i think i can set some tag for all elements in the group  to select them when setting up the style, but i cant place !elemets to my diagram code.

How to use it?)

**Comment by simonbrowndotje** [comment]
Should be released later today -> https://github.com/structurizr/java/blob/master/changelog.md
